packages/progress-bar/src/ProgressBar.ts

```diff
@@ -99,15 +115,14 @@ export class ProgressBar extends SizedMixin(SpectrumElement) {
             if (this.indeterminate) {
                 this.removeAttribute('aria-valuemin');
                 this.removeAttribute('aria-valuemax');
+                this.removeAttribute('aria-valuenow');
```

Comment:
where are  you adding this attribute?

Reply:
right below these lines... when the changes include progress and indeterminate is false